doc/classes/String.xml

```diff
@@ -448,14 +473,14 @@
 			<argument index="0" name="position" type="int">
 			</argument>
 			<description>
-				Return an amount of characters from the left of the string.
+				Returns an amount of characters from the left of the string.
 			</description>
 		</method>
 		<method name="length">
 			<return type="int">
 			</return>
 			<description>
-				Return the length of the string in characters.
+				Returns the length of the string in characters.
```

Comment:
is this the amount of characters? could be a bit more precise differently because "in characters" sounds like the returntype would be characters
suggestion:
"Returns the string's amount of characters"